recipes/netcdf-cxx-legacy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    export DYLD_LIBRARY_PATH=${PREFIX}/lib
+fi
+
+CPPFLAGS=-I$PREFIX/include LDFLAGS=-L$PREFIX/lib ./configure --prefix=$PREFIX
```

Comment:
You are stomping on the built in compiler flags here. Better to append.

Reply:
Especially since the LDFLAGS set rpaths.

Reply:
I may not need these at all.  I copied them from a rather early version of the `netcdf-cxx4` feedstock before that one switched to CMake.  I'll try taking them out.